### DISCUSSION TITLE
Suddenly send ON command

### DISCUSSION THREAD

**Discussion by thrahlf** [Koenkk/zigbee2mqtt#15163] [Devices]
Hi,

lately a device gets an ON command every night (around 04:00) without there being a rule. Then the light goes on...

Has this ever happened to anyone and maybe has a solution?

Is it otherwise also possible to deactivate the device explicitly in a period? I have also already re-taught it, but this has brought nothing. Unfortunately...

Regards,
xtoasty

**Comment by lux73** [comment]
i have strong doubts that this is initiated from z2m - please review all Rules & Automations on your Home Automation Software!
